### PR TITLE
AP_HAL_ChibiOS: add hwdef MFE_POS3_CAN

### PR DESCRIPTION
**Pull request by mikefenghao**
This is the hwdef file for MFE_POS3_CAN. We've done a lot of testing and it's working fine.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# MCU class and specific type
+MCU STM32F4xx STM32F427xx
+
+FLASH_RESERVE_START_KB 36
+
+STORAGE_FLASH_PAGE 16
+define HAL_STORAGE_SIZE 8192
+
+# board ID for firmware load
+APJ_BOARD_ID 6101
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat:15]: use logical id

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat (hunk 1 above)
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat:15]: You need to reserve this board ID in `Tools/AP_Bootloader/board_types.txt` and then use the name you use in that file in here.

**Review by peterbarker — DISMISSED** [review]
Grab 10 IDs in board_types.txt while you're there - you can see how others are doing that in there.

File: libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat (hunk 1 above)
Comment (tridge) [reply libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat]: fixed

**Review by tridge — APPROVED** [review]

**Pull request merged**